content/ja/docs/concepts/workloads/controllers/job.md

```diff
@@ -0,0 +1,888 @@
+<!--
+reviewers:
+- erictune
+- soltysh
+title: Jobs
+content_type: concept
+feature:
+  title: Batch execution
+  description: >
+    In addition to services, Kubernetes can manage your batch and CI workloads, replacing containers that fail, if desired.
+weight: 50
+-->
```

Comment:
+1
Usually, I check translations between original contents and translated one on side-by-side by myself.
And in this scheme, we need to check that all of comment outs are removed on last phase.

Reply:
Well noted and thank you so much for the information! Is it ok if I leave it there during the hold reviewing process and remove them when this PR is ready to merge? I think it will be easier for you to review.

Reply:
we need to do re-review when you make "change" for the PR content so it's better to remove when the PR is ready for review.

Reply:
That's true. I'll go revise it. Thank you for your kind instruction!